src/widgetastic_patternfly/__init__.py

```diff
@@ -669,17 +670,30 @@ def is_opened(self):
     def is_closed(self):
         return not self.is_opened
 
+    def click(self):
+        """Override Clickable's click."""
+        self.browser.click(self.HEADER_LOCATOR)
+
     def open(self):
         if self.is_closed:
             self.logger.info('opening')
             self.click()
-            # Workaround for stupid Settings page, perhaps we put a try mechanism in here
-            if self.is_closed:
-                self.click()
+            try:
+                wait_for(lambda: self.is_opened, delay=0.1, num_sec=3)
+            except TimedOutError:
+                self.logger.warning('Could not open the accordion, trying clicking again')
+                # Workaround stupid pages, perhaps we put a try mechanism in here
+                if self.is_closed:
+                    self.click()
+                    try:
+                        wait_for(lambda: self.is_opened, delay=0.1, num_sec=3)
+                    except TimedOutError:
+                        self.logger.error('Could not open the accordion')
```

Comment:
I did this specifically to not have that exception printed out, because we know why that happens.